src/Comparator/BackwardsCompatibility/InterfaceBased/InterfaceBecameTrait.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Roave\ApiCompare\Comparator\BackwardsCompatibility\InterfaceBased;
+
+use Assert\Assert;
+use Roave\ApiCompare\Change;
+use Roave\ApiCompare\Changes;
+use Roave\BetterReflection\Reflection\ReflectionClass;
+use function sprintf;
+
+/**
+ * An interface cannot become a trait without introducing an explicit BC break, since
+ * all implementors need to be changed to implement it instead of extending it.
+ */
+final class InterfaceBecameTrait implements InterfaceBased
+{
+    public function compare(ReflectionClass $fromClass, ReflectionClass $toClass) : Changes
+    {
+        Assert::that($fromClass->getName())->same($toClass->getName());
+
+        if (! $toClass->isTrait() || ! $fromClass->isInterface()) {
+            // checking whether an interface became an class is done in `InterfaceBecameClass`
+            return Changes::new();
+        }
+
+        return Changes::fromArray([Change::changed(
+            sprintf('Interface %s became a trait', $fromClass->getName()),
+            true
+        ),
```

Comment:
Something weird causing these alignment issues in phpcs I expect...

Reply:
Verified this: it's an issue in one of the sniffs.

Basically, single-value array should be on a single line, but then the value turns out to be multi-line here. Trying to fix it leads to a phpcs violation being reported, so I'll keep these wonky ones as they are here, and talk to the doctrine folks about it.